### DISCUSSION TITLE
How to filter the options for select?

### DISCUSSION THREAD

**Discussion by bombillazo** [SBoudrias/Inquirer.js#1387] [General]
We have a long list of items and would like users to type and search to eliminate options

**Comment by SBoudrias** [comment]
Hi, this isn't supported natively in the default prompts (and there's no plan to add it.)

The approach for a prompt like this is to implement a custom one. The [API documentation is over here](https://github.com/SBoudrias/Inquirer.js/tree/master/packages/core), and our [testing utilities here](https://github.com/SBoudrias/Inquirer.js/tree/master/packages/testing).
